src/vic3save/src/vic3date.rs

```diff
@@ -91,6 +91,16 @@ impl Vic3Date {
     pub fn hour(&self) -> u8 {
         self.raw.hour()
     }
+
+    pub fn add_days(self, days: i32) -> Vic3Date {
+        let new_date = Date::from_ymd(self.year(), self.month(), self.day()).add_days(days);
```

Comment:
No need to do anything, but this makes me realize that `RawDate` should be the one to have `add_days`

Reply:
yeah, but I was super happy to find this is already implemented. Unfortunately it was after I've also implemented it xD (thus the test cases).